plugin/manager_linux.go

```diff
@@ -105,6 +109,7 @@ func (pm *Manager) pluginPostStart(p *v2.Plugin, c *controller) error {
 		}
 
 	}
+	p.SetAddr(&net.UnixAddr{Net: "unix", Name: sockAddr})
```

Comment:
Can we set this before L76, so that we can do `NewClientWithTimeout(Addr(),...)`  ?